lib/getters/amplienceGetters.ts

```diff
@@ -0,0 +1,148 @@
+import getConfig from 'next/config'
+
+const { publicRuntimeConfig } = getConfig()
+
+const formatLinkData = (links: any) => {
```

Comment:
are these getters still needed? if so we should move under the `/cms` folder to keep everything togeter

Reply:
Done